auditing_standards.md

```diff
@@ -218,15 +218,25 @@ Crates with this criteria contain unsafe Rust code which doesn't uphold the typi
     *   Writing implementations of traits not marked `unsafe` by violating documented invariants.
     *   Writing implementations of traits not marked `unsafe` that are not really intended to be implemented by the user.
     *   Explicitly forgetting values that have important drop behavior to cause UB when combined with operations that would not be expected to follow normally.
-*   A fair amount of caution may be required to avoid undefined behavior.
+*   Some caution may be required to avoid undefined behavior.
+
+
+`ub-risk-3` is expected to live at the threshold of acceptance as a dependency for most codebases: there is unsoundness in the crate, but there are also many situations where one need not worry about that unsoundness ever being triggered, and the codebase needs to make a judgement call. For example, one may choose to depend on a `ub-risk-3` dep if the rating is due to:
+
+ * Unsoundness that is only triggered on platforms that the codebase does not care about.
```

Comment:
```suggestion
 * Can only be triggered on unsupported targets.
```

Reply:
I would say "targets the codebase doesn't use" or "unused targets". The targets are *supported*, in that the crate will presumably build for and run on them.